### ISSUE TITLE
Get (inverse) hessian sub-matrices

### ISSUE THREAD

**Issue by eweine** [yixuan/LBFGSpp#43]
Related to https://github.com/yixuan/LBFGSpp/issues/42, would it be possible to retrieve the matrices approximating the (inverse) hessian once optimization has terminated? I believe this will be a low-rank approximation.

Thanks,

Eric.

**Comment by yixuan** [comment]
Yeah, this has been done in https://github.com/yixuan/LBFGSpp/commit/f800f188690b8df08056326c7f3bccf48e361246. Any feedback is welcome!
